aws/Dockerfile

```diff
@@ -2,4 +2,8 @@ ARG REPOSITORY_BASE_PATH
 
 FROM ${REPOSITORY_BASE_PATH}:latest
 
-RUN pip install boto3 botocore
+USER root
```

Comment:
I'm not sure we need those user directives.
The default user was root before, we should not switch it to spacelift as it will introduce BC breaks.

Reply:
nope, it's always been spacelift

Reply:
~~Isn't the `aws` and `gcp` flavors inherit the `base` one?~~

Ha fuck nevermind, I've missed the `USER spacelift` in the `base` dockerfile 🤦🏻